app/utils.py

```diff
@@ -82,3 +85,45 @@ def fetch_cases_by_date(case_date_from, case_date_to):
                     'signed_date': contract['ActivatedDate'],
                 })
     return result
+
+
+def generate_presto_query(event_id=None, from_date=None, to_date=None, currency=None):
+    currency = currency or 'BRL'
```

Comment:
What if you put the default value of currency in the method definition?  ```python
def generate_presto_query(event_id=None, from_date=None, to_date=None, currency='BRL')
```

Reply:
I tried that before but when no currency is provided, the parameter takes the value None instead of the default value. I think putting the value in the definition only works when no argument is provided when calling the function.